internal/telemetry/event.go

```diff
@@ -30,10 +30,10 @@ const (
 	EventTypeCommandError    EventType = "COMMAND_ERROR"
 )
 
-// EventDataKey is an event data key
-type EventDataKey string
-
 // set of event data keys
 const (
-	EventDataKeyErr EventDataKey = "err"
+	eventDataKeyCommand     string = "cmd"
```

Comment:
[nit] shouldn't need the `string` types here (the `const` will pick that up automatically)